### DISCUSSION TITLE
Missed readin alert

### DISCUSSION THREAD

**Discussion by Lyver29** [NightscoutFoundation/xDrip#2277] [General]
I have a problem with Missed reading alert which is not triggered at all. Once it just stopped working. Please, help me to solve this. I'm xDrip follower, Missed reading alert is on, volume is on. There is no alert in the logs. All other alerts work good.
![inbound6805430109203808178](https://user-images.githubusercontent.com/111521089/185432754-becafcd6-f6c9-4adb-8af8-7b7ba836c564.jpg)
![inbound9168422405628355126](https://user-images.githubusercontent.com/111521089/185432929-5514989f-8837-4c53-879d-dd9b510c4c57.jpg)
![inbound8735541504618466309](https://user-images.githubusercontent.com/111521089/185432965-81dbce15-790c-4c4f-aed8-3666fb322024.jpg)
![inbound3582269630698706082](https://user-images.githubusercontent.com/111521089/185432977-b8a14df8-b631-4b1a-8091-d46c3a49f26a.jpg)

**Comment by tzachi-dar** [comment]
Did you reboot your phone in the last 15 minutes? If yes, this might explain it.
What phone do you have? Did you remember to get xDrip out of the battery optimizations and all kind of (deep) sleeping mode.

**Reply by Lyver29** [reply]
No, last time i reboot my phone was yesterday. It is Samsung Galaxy M12, Android 11, battery optimization is off.

**Comment by Navid200** [comment]
Can you also show the "Followers" status page?  Assuming there is no private ID on it.

**Comment by tzachi-dar** [comment]
Can you please go to settings->less common settings->extra logging settings
and on enter more tags to be logged enter: AlertPlayer:v,MissedReadingService:v
Next when there should be an alert and there isn't one, please take a screenshot of the logs like you have before and post it here.
